### DISCUSSION TITLE
Diferentes resoluções de imagens para app mobile

### DISCUSSION THREAD

**Discussion by laurindo** [frontendbr/forum#872] [Perguntas e Respostas]
Pessoal, vocês conhecem alguma ferramenta onde eu possa testar se a resolução da imagem para dispositivos moveis irá se comportar bem?
Estou usando umas imagens de introdução no meu aplicativo mobile(React Native). Porem eu preciso que o tamanho do apk final não fique grande por conta de algumas imagens, e como eu tenho imagens em alta resolução isso pode deixar o apk bem grande.

Desde já agradeço.

**Comment by letanure** [comment]
Eu resolvi usar cloudinary para meu sistema legado, com upload pela url mesmo, 

ele gera o tamanho exato para cada dispositivo com um tamanho até 70% menor, fora gerar .webp etc

**Comment by laurindo** [comment]
Hmm, achei interssante. vlw @letanure . Vou testar.

**Comment by felipefialho** [comment]
A questão é o tamanho do apk e não o carregamento em si?

**Comment by laurindo** [comment]
@LFeh Meu medo é jogar imagens em diferentes resoluções e o APK ficar grande.

**Comment by felipefialho** [comment]
Entendi, nesse caso acho que a solução do @letanure pode cair bem.
